catalog/task/collect-data/0.1/collect-data.yaml

```diff
@@ -38,7 +38,7 @@ spec:
       description: The first component's containerImage in the snapshot to use in the fbc pipelines
   steps:
     - name: collect-data
-      image: quay.io/hacbs-release/release-utils:f486f350df43974706fc484a13bfeefc2c8fa95a
+      image: quay.io/hacbs-release/release-utils:4d8649dbb2b626f5fe9f4ff83c1bc3be268fad31
```

Comment:
Why was this changed? There's nothing about it in the commit message. Is there a new functionality that is needed? If so, please link the corresponding PR. Or is it just to be on the latest image? That's fine, but please mention it in the commit message.

Reply:
I updated the commit message. The new thing is https://github.com/redhat-appstudio/release-service-utils/pull/48

Reply:
Cool, thanks.